decidim-core/app/cells/decidim/tags_cell.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This cell renders the category of a resource
+  # shown with the translated name and links to
+  # the resource parent `feature` and `parcticipatory space` index.
```

Comment:
`s/parcticipatory/participatory/`

Reply:
yes, I'll fix it, I'll also `s/feature/component/`